types/src/toGamut.d.ts

```diff
@@ -1,24 +1,45 @@
 import { ColorTypes, PlainColorObject } from "./color.js";
+import { Methods } from "./deltaE/index.js";
 import ColorSpace, { Ref } from "./space.js";
 
+export interface Options {
+	/**
+	 * How to force into gamut.
+	 *
+	 * If `"clip"`, coordinates are just clipped to their reference range.\
+	 * If `"css"`, coordinates are reduced according to the CSS 4 Gamut Mapping Algorithm.\
+	 * If in the form `[colorSpaceId].[coordName]`, that coordinate is reduced
+	 * until the color is in gamut. Please note that this may produce nonsensical
+	 * results for certain coordinates (e.g. hue) or infinite loops
+	 * if reducing the coordinate never brings the color in gamut
+	 */
+	// eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
My assumption is that @LeaVerou would prefer that we adjust the lint rule rather than overriding it in the codebase. If that's the case, we could remove this and use the following for the rule in `.eslintrc.cjs` to allow `{}` types:

```js
		"@typescript-eslint/ban-types": [
			1,
			{
				"extendDefaults": true,
				"types": {
					"{}": false,
				},
			},
		],
```

Reply:
IMO it might be worth keeping it as a comment to just disable it for this line, since there are scenarios where `{}` might accidentally end up used where it shouldn't be (like somebody accidentally using it where they meant to have a type that accepts any object). If there were an eslint rule to allow specifically `string & {}` just for this use case then that would be ideal, but I don't know whether that's actually possible.

We can wait to see if @LeaVerou has thoughts on which approach is better

Reply:
@jgerigmeyer is right. (I should write up about this somewhere to explain my reasoning so it doesn't just live in random PRs)